### DISCUSSION TITLE
FBA Inbound API v2024-03-20 - Proper way to set Prep Types?

### DISCUSSION THREAD

**Discussion by MetaMan2023** [amzn/selling-partner-api-models#4450] [Q&A]
We are having trouble properly setting prep categories and types.  The new API method seems a little clunky and inconsistent to the real world. 

Some examples:

* We have found if you choose a prep category like "LIQUID", Amazon "requires" you to either choose NO_PREP or three very specific prep requirements: Polybag, Cap Sealing, and Suffocation warning.   This is a strange combination -- not all liquids need cap Sealing prep (they may come compliant from the factory).   Also Suffocation warning stickers are only required for bags > a certain opening size, so this seems to be a strange requirement.

NOTE: This issue is consistent for other "categories" as well.

* The other oddity is if you choose "NO_PREP", you can literally choose anything else... for example I can say:  CATEGORY = LIQUID, PREP TYPE = NO_PREP, POLYBAG, LABEL.     This also seems strange, as "no prep" would seem to indicate "no prep required" -- yet I can still list other prep.

It seems very hard to set "real world" requirements in the new prep tool -- are we misunderstanding the process?

* There are two confusing "sold as set" options as well -- they are very similarly labeled and we are not really sure what the difference is.  Also, would any bundle or multi-pack be "sold as set" no matter how it's packed (Polybag or box) -- we understand sold as set to be for things like a master case of widgets where you are selling a full case that contains 24 individual items, so the WH does not open the box by accident.   But a polybag that may be a bundle of two items would that also be considered a set?

If a polybag is considered a set, then it would seem that all polybags should be labeled sold as set -- as there would be no way for the FBA center to tell what is inside the polybag without opening it.

Looking for guidance on this.

**Comment by JimMcKenzieSmith** [comment]
I agree that we need some clear guidance here.  Otherwise we will be reverting to wonky workarounds (already happening).  

I would also like to see clear guidance and documentation on how these settings related to prepOwner and labelOwner... right now the logic seems to be a mess.
